### PR TITLE
fix(modal): don't scale modal with size="zen"

### PR DESCRIPTION
**Pull request by denkristoffer**
# Purpose of PR

### CODE REVIEW

**Review by Lelith — APPROVED** [review]
Looks good to me 👍  
But this might be a really stupid question, why is it named zen? as like in "zen buddhism"?

**Comment by denkristoffer** [comment]
>  But this might be a really stupid question, why is it named zen? as like in "zen buddhism"?

Great question! I don't know, it was before my time 😆 I've always assumed it was about providing a zen state, something meditative, uninterrupted or focused.

File: .changeset/long-onions-behave.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@contentful/f36-modal": patch
+---
+
+fix(modal): fix scalign issues for modals with size "zen"
```
Comment (massao) [inline .changeset/long-onions-behave.md:5]:
```suggestion
fix(modal): fix scaling issues for modals with size "zen"
```

**Review by massao — APPROVED** [review]
Small typo but looks good

**Pull request merged**